scripts/west_commands/tests/test_build.py

```diff
@@ -4,9 +4,10 @@
 
 from argparse import Namespace
 
-from build import Build
 import pytest
 
+from build import Build
```

Comment:
I think there's something wrong here because `build` is an actual python package instead of targeting the local one, which we are doing here.

The same mess is for the `conftest` imports.

I need to figure out what is going on here, but I'll do it later and exclude them for now.